### DISCUSSION TITLE
Group 4: The effectiveness and economic impacts of cap-and-trade programs in the U.S.

### DISCUSSION THREAD

**Discussion by bbdaniels** [gui2de/ppol768-spring23#160] [Group Projects Feedback]
Group 4, this is a super interesting question and we love to see some large-scale (ie, non-survey / non-individual) questions! Feedback follows:

Market failure is, I guess, an easy one here, so unlike the groups that are dealing with survey and individual level data, you don't have much to justify here. The climate stuff is mostly very well known and established and you should not spend a lot of time on this project rehashing this or the theory of change, as that would be literature review or copy-paste which we want to avoid. So in that sense you have picked a great project; on the other hand, you should expect to spend most of your time putting thought and effort into the design and measurement elements for this course.

This, however, is a great challenge, as it will very much direct your thinking to how microdata and impact evaluation can link at aggregate scales, which gives you many options. First, for example, I notice you have restricted your sample to "facilities". Is this population easy to define? How might it differ from "companies", which may for example have the ability to "near-shore" their facilities to states or countries (Canada/Mexico in particular) that are not covered by the two programs you have identified? What happens if facilities open and close over the course of your study period? These are all important questions that you will need to lock down as you advance the program, but all of them give you great avenues of approach.

You might consider broadening the set of controls. You say "First, XXX, one of the rest of states having not passed a cap-and-trade law of any design, enables us to answer the first research question". Why not use all the remaining states? What about neighboring countries (or comparable countries, since California is country-sized)? Should you pool the RGGI states into a "region", or should they really be treated separately? Methods like "synthetic controls" have been developed to compare, for example, California tobacco policy to all other 49 states in the past, since it is a serious challenge to have a "treatment group" that only has 10 observation points (2013-2023) -- and you will see later in the class's regression/identification series that sampling more facilities or companies from the same macro environment does not always help with this problem.

Finally, of course, there is data. While we don't expect the data collection plans to be 100% plausible (ie, don't worry about, say, funding constraints), they need to be feasible. I do not think it is likely that you will get very good response to "a self-reported survey of emissions, profit, and employment" through a voluntary facility-level survey! Companies lie to the **government** about this stuff! Secondly, while it is a novel research idea and we applaud the innovation, we cannot allow time travel as a research method. If you want to know about the past, you can either ask retrospective surveys or find another source of data... this is a major research constraint always, since if we knew what we wanted to study in the future we would start surveying it now.... There have been very many "lucky" studies that only happened because a war broke out or a disaster happened right in the middle of a _different_ project. This isn't one of them, so you will have to grapple with your sources of retrospective data.

That all said, the project idea is perfect for this class and I am super supportive of it, since is gives unique challenges to your group to solve in terms of comparison, measurement, and design strategy. Looking forward to seeing where you take it!

**Comment by BeaLeydier** [comment]
Hi team!

Seconding Ben's comments that it is nice and refreshing seeing a macro project on a well studied topic, because it allows (forces!) you to be original in the design and measurement aspects.

Your problem statement and research questions are very clear, which is great. One thing that came out of the blue in the ToC is the mention of employment - it's not very clear to me how that's related to anything else or to the cap and trade scheme directly. So I would suggest to either expand on that (why that is related, and why that is important for you to measure; maybe there's a trade-off between employment and emissions and that is an aspect you want to study) or drop it and focus on the emissions.

To measure emissions, surveys are tricky (and especially in the past as Ben noticed!), but there are a lot of other related ways : can you look at observational data, like air quality? satellite imagery? temperatures? seasonal changes? Because that's a well studied topic, the literature here will be helpful to find ideas on how what data can give you a sense of emissions and other outcomes you are looking at. 

For cross-border policy changes, it's also common to look at study actors (whether you define them as firms, people, cities, or anything else) just across both sides of the border, to see how trends differ after the policy change. You would also want to be able to establish that trends were similar prior to the policy change; and that there aren't selection issues generated by the policy change itself (like the firms closure or the delocalisations Ben mentions).

Another thought that came to mind in comparing the cross-sector vs the power sector cap and trade is the idea of crowd-out : because firms interact with each other, it will be interesting to think about how the power sector impacts other sectors in places with and without cap and trade. For example, if energy becomes very expensive, it will affect other sectors in ways that may have emissions impact as well.

Cheers!

**Comment by bbdaniels** [comment]
# Concept Note Review

Hi team, this is fantastic work and a very thoughtful extension and improvement of the pitch. Great job! I am assuming a few things that you should make more clear. First -- you are using "facilities" as the unit of observation, right? Second -- it would be really nice if you could include a diagram showing exactly the timeline for all the states you use and which you are comparing to which (when) -- this will help the reader understand the approach much better. I very much like the addition of the New Jersey and the AZ/NV/OR comparisons; this makes like four groups with different timelines and I would like to see the exact comparisons laid out visually (I think they are there in the concept).

Your MAJOR problem is the super-high correlation in treatment status. All treatment facilities (CA) have exactly the same operating environment. All control facilities (RGGI) have exactly the same operating environment. This means you are going to have power/clustering problems. These problems are made even worse by the fact that you have a wide range of potentially substitutable output pollutants. Which are controlled by the relevant cap-and-trade facilities? Which are not? How can you account for multiple outcomes (Bonferroni and Benjamini-Hochberg approaches)?

We will cover more of this next week, but the core question for you will be -- is it statistically feasible to detect relevant-sized changes in emissions using this design? This is not at all a weakness of the project, by the way, even if power is very low -- this is exactly where we want you to be so you can assess this coherently in the final draft. But it might sink the project _in reality outside the classroom_. What _are_ relevant-sized changes in emissions? Of one pollutant? Of all of them? What are you able to detect? Taking seriously those parameters and building that out in the final version by doing simulated approaches to your modeling along the lines of what we are covering in my lectures will make this an excellent final submission. Keep it up!
